src/Microsoft.TestPlatform.Utilities/CodeCoverageDataAttachmentsHandler.cs

```diff
@@ -35,93 +33,68 @@ public IEnumerable<Uri> GetExtensionUris()
             yield return CodeCoverageDataCollectorUri;
         }    
 
-        public Task<ICollection<AttachmentSet>> ProcessAttachmentSetsAsync(ICollection<AttachmentSet> attachments, IProgress<int> progressReporter, IMessageLogger logger, CancellationToken cancellationToken)
+        public async Task<ICollection<AttachmentSet>> ProcessAttachmentSetsAsync(ICollection<AttachmentSet> attachments, IProgress<int> progressReporter, IMessageLogger logger, CancellationToken cancellationToken)
         {
             if (attachments != null && attachments.Any())
             {
                 var codeCoverageFiles = attachments.Select(coverageAttachment => coverageAttachment.Attachments[0].Uri.LocalPath).ToArray();
-                var outputFile = MergeCodeCoverageFiles(codeCoverageFiles, progressReporter, cancellationToken);
+                var outputFile = await this.MergeCodeCoverageFiles(codeCoverageFiles, progressReporter, cancellationToken);
```

Comment:
in await lines always add .ConfigureAwait(fales) at the end

Reply:
Done.